### PR TITLE
158502436-Added tests and readme badges

### PR DESCRIPTION
**Pull request by kwanj-k**
### What does this PR do?
Adds tests for the API end-points

### Description of Task to be completed
* Create failing tests for all API endpoints and run them with pytest 

### How should I test it?
 * Clone this repository to a local folder
 * Navigate to the cloned repo and Run:
 * virtualenv -p python3 . -to create virtual env 
 * source bin/activate -activate virtualenv
 * pip install requirements.txt
* pytest

### What PT stories?
[ft-tests-#158502436]https://www.pivotaltracker.com/story/show/158502436

### CODE REVIEW

**Comment by TheSteelGuy** [comment]
were we to  really create another repo for challange2, coz i think though the they are different challanges the product is one.

**Comment by JoshuaOndieki** [comment]
Write your commits in the imperative mode.
See here :arrow_right: https://github.com/erlang/otp/wiki/writing-good-commit-messages for tips on writing a good commit message.

File: tests/test_auth.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+"""Tests associated with authenitication."""
+import unittest
+import json
+
+from app import create_app
+
+config_name = "testing"
+app = create_app(config_name)
```
Comment (JoshuaOndieki) [inline tests/test_auth.py]: Initialize your app instance inside the setUp() method

File: tests/test_auth.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            data=json.dumps(self.data),
+            content_type='application/json')
+        res = self.app.post(
+            "/api/v1/auth/signup",
+            data=json.dumps(self.data),
+            content_type='application/json')
+        #response = json.loads(res.data.decode("utf-8"))
+        self.assertEqual(res.status_code, 409)
+        #self.assertEqual(response["status"], "Failed!")
+        #self.assertEqual(response["message"], "User already exists!")
```
Comment (JoshuaOndieki) [inline tests/test_auth.py]: Remove unused code

File: tests/test_ride.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
+"""Tests associated with Rides"""
+import unittest
+import json
+
+from app import create_app
+
+config_name = "testing"
+app = create_app(config_name)
```
Comment (JoshuaOndieki) [inline tests/test_ride.py]: Initialize your app instance inside the setUp() method

**Comment by kwanj-k** [comment]
Latest branch was merged to develop

**Pull request closed without merging**